_posts/2024-04-18-hpc-env-setup.md

```diff
@@ -0,0 +1,46 @@
+# Using HPC to Run Scripts
```

Comment:
Each post needs a YAML header. Take a look at the other posts (or template) for what needs to be added.